cmd/erasure-object.go

```diff
@@ -1029,7 +1030,7 @@ func (er erasureObjects) DeleteObjects(ctx context.Context, bucket string, objec
 
 			// all other direct versionId references we should
 			// ensure no dangling file is left over.
-			er.addPartial(bucket, version.Name, version.VersionID)
+			er.addPartial(bucket, version.Name, version.VersionID, 0)
```

Comment:
```suggestion
			er.addPartial(bucket, version.Name, version.VersionID, -1)
```

Should be set to `-1` to indicate delete? since fi.Size can be `0` for 0 byte objects.